script/Arcueid/Day 4/04_00_ARC04_6_1.txt

```diff
@@ -391,7 +391,7 @@ are all spinning like an emergency light.
 [sha:317bb9f7f0e64fb7534ddb26669cae351ffb2720]{
 -- Page 932, Offset 8137.
 -- ごうごう。ごうごう。ごうごう。
-%{i}Rumble. Rumble. Rumble.%{/i}
+%{i}Rumble%{/i}. %{i}Rumble%{/i}. %{i}Rumble%{/i}.
```

Comment:
you can just put this whole line in italics

Reply:
whoops my bad